PORTAL/jobs/_pyperformance.py

```diff
@@ -407,7 +406,7 @@ def build_compatid(
     @classmethod
     def normalize_suite(cls, suite: SuitesType) -> SuiteType:
         if not suite:
-            return cls.SUITE_NOT_KNOWN
+            return cls.MULTI_SUITE
```

Comment:
The initial results file is `unknown` based on the filename.  However, when we use `PyperfUploadID.from_metadata()` (e.g. in `PyperfResults.uploadid`), we do get `multi`.  If that isn't getting triggered then therein lies our bug. :)

Reply:
Where is `from_metadata` supposed to get triggered from?  I can't see how to get [this line to run](https://github.com/faster-cpython/tools/blob/main/PORTAL/jobs/_pyperformance.py#L1504) because `self.resfile.uploadid` is always non-None when you load it from a file.